dist/spec.json

```diff
@@ -13555,6 +13793,159 @@
         "Policy"
       ]
     },
+    "Domain": {
+      "properties": {
+        "certificateSourcetype": {
+          "enum": [
+            "MANUAL"
+          ],
+          "type": "string"
+        },
+        "domain": {
+          "type": "string"
+        }
+      },
+      "type": "object",
+      "x-okta-crud": [
+        {
+          "alias": "delete",
+          "arguments": [
+            {
+              "dest": "domainId",
+              "src": "id"
+            }
+          ],
+          "operationId": "deleteDomain"
+        },
+        {
+          "alias": "read",
+          "arguments": [
+            {
+              "dest": "domainId",
+              "src": "id"
+            }
+          ],
+          "operationId": "getDomain"
+        }
+      ],
+      "x-okta-operations": [
+        {
+          "alias": "verify",
+          "arguments": [
+            {
+              "dest": "domainId",
+              "src": "id"
+            }
+          ],
+          "operationId": "verifyDomain"
+        }
+      ],
+      "x-okta-tags": [
+        "Domain"
+      ]
+    },
+    "DomainCertificate": {
+      "properties": {
+        "certificate": {
+          "type": "string"
+        },
+        "certificateChain": {
+          "type": "string"
+        },
+        "privateKey": {
+          "type": "string"
+        },
+        "type": {
+          "enum": [
+            "PEM"
+          ],
+          "type": "string"
+        }
+      },
+      "x-okta-tags": [
+        "Domain"
+      ]
+    },
+    "DomainCertificateMetadata": {
+      "properties": {
+        "expiration": {
+          "type": "string"
+        },
+        "fingerprint": {
+          "type": "string"
+        },
+        "subject": {
+          "type": "string"
+        }
+      },
+      "x-okta-tags": [
+        "Domain"
+      ]
+    },
+    "DomainLinks": {
+      "properties": {
+        "certificate": {
+          "type": "object"
+        },
+        "self": {
+          "type": "object"
+        },
+        "verify": {
+          "type": "object"
+        }
+      },
+      "x-okta-tags": [
+        "Domain"
+      ]
+    },
+    "DomainListResponse": {
+      "properties": {
+        "domains": {
+          "items": {
+            "$ref": "#/definitions/DomainResponse"
```

Comment:
for `DomainListReponse` items, I think we will need to have a separate definition for the array item. 

It is currently defined as` DomainResponse` which contains all the domain properties and is returned when we do a `Get` for single domain. 
Whereas for `Get All` we will respond with a subset of properties for each domain - https://developer.okta.com/docs/reference/api/domains/#domainlistresponse-properties

Reply:
Please ignore above comment, I think it looks correct as it is aligned with the implementation. I will try to update the docs to reflect the same.